packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/KycVerification/UserInfoDetails/selectors.ts

```diff
@@ -0,0 +1,32 @@
+import { compose, lift } from 'ramda'
+
+import { Remote } from '@core'
+import { model, selectors } from 'data'
+import { RootState } from 'data/rootReducer'
+
+const { USER_INFO_DETAILS } = model.components.identityVerification
+
+// remove unused stuff from userDetails
+/* eslint-disable */
```

Comment:
This is purely personal - but can we denote _what_ is being disabled? I'm guessing here is just the unused params - but being specific reduces the chance of making another lint error here.